docs/source/models/cause.rst

```diff
@@ -558,6 +558,53 @@ below.
 Remission Rates
 ^^^^^^^^^^^^^^^
 
+**Definition**
+
+Remission is a measure of cases that recover from a with-condition state, given 
+a specified population and time period. Just as with incidence, the numerator is 
+given by the count of recovered (or *remitted*) cases, and the denominator is 
+the cumulative person-time [link to person-time definition in incidence 
+section] during which cases are able to go into remission.
+
+  For example, consider diarrhea cases in the Philippines in 2017. Say there
+  were 1 million prevalent cases of diarrhea over the course of 2017, and each
+  case remitted after an average of 5 days. The way we define this remission
+  rate depends on the denominator we choose; some examples include:
+
+.. math::
+	
+	\frac{\text{1/5 case}}{\text{1 person-days}}=\frac{365/5\text{ cases}}{\text{1
+   person-year}}=73\text{ cases/person-year}
+ 
+Note that, unlike incidence, there is not a strong precedent for referring to 
+remission rates within the entire population; rather, remission rates typically 
+refer to rates within an infected, or with-condition population. {Is this true? 
```

Comment:
My understanding was that this is also usually true of incidence rates, and using total population in the denominator is a quirk of GBD (or is used an approximation when more detailed data isn't available). But I could be wrong.

Reply:
Agreed!